src/gordon_gcp/plugins/service/event_consumer.py

```diff
@@ -143,6 +144,15 @@ def _validate_config(self):
                    'Cloud Pub/Sub is required.')
             errors.append(msg)
 
+        if 'max_msg_age' not in self.config:
+            self.config['max_msg_age'] = 300
+
```

Comment:
Any reason for doing this here?  I wouldn't expect a method called validate to change the state of anything.
I would more expect a default to be set in the consumer init.

Reply:
I mean, this is how we've done it elsewhere, but...

Reply:
Also, this centralizes all the config processing.  Unless you want me to add 'apply_defaults()'.

Reply:
Spoke to @descent3me, moving it as requested.